bitbots_odometry/src/odometry_fuser.cpp

```diff
@@ -27,21 +29,34 @@ class OdometryFuser {
   ros::Time _imu_update_time;
   ros::Time _odom_update_time;
   geometry_msgs::TransformStamped tf;
+  char current_support_state_;
+  tf2_ros::Buffer tf_buffer_;
+  tf2_ros::TransformListener tf_listener_;
```

Comment:
If I see it correctly the constructor will leave some of the variables memory in an undefined state.

Reply:
no they are initialized in constructor